openedx/features/qverse_features/registration/signals.py

```diff
@@ -95,7 +100,8 @@ def _create_or_update_edx_user(user_info):
 
     edx_user, created = User.objects.update_or_create(username=user_info.get('regno'), defaults=user_data)
     if created:
-        edx_user.set_password(user_info.get('regno'))
+        user_info['password'] = get_random_string()
+        edx_user.set_password(user_info['password'])
```

Comment:
```suggestion
        edx_user.set_password(get_random_string())
```

Reply:
We will be sending a registration email to each student with their login credentials. For that, we need to save `password` somewhere. This is the step where we are doing that thing.